### ISSUE TITLE
Table not rendering correctly when using PageDown/Up keys

### ISSUE THREAD

**Issue by jiteshvassa** [handsontable/handsontable#2664]
Using scroll bars in the example below works as expected.  However when using the PageDown key the lower section of the table doesn't get rendered.

Example here: http://jsfiddle.net/27jxhp7w/

**Comment by krzysztofspilka** [comment]
@jiteshvassa could provide us with more details on this? I followed to your demo and it seems to be working just fine.

**Comment by jiteshvassa** [comment]
Thanks for the quick response!  I've attached two screenshots:

scrollbar.jpg: shows the behavior when using the scrollbar (which looks fine).
pagedown.jpg: shows the behavior when using the pagedown key, you can see the lower part of the table seems to have not rendered.

I'm wondering whether it may be a browser related issue if you're unable to replicate it.  I'm using Chrome Version 43.0.2357.134 m on Windows 7.

![scrollbar](https://cloud.githubusercontent.com/assets/5113964/8823855/48eba222-3068-11e5-82df-d6b20bc1e442.jpg)

![pagedown](https://cloud.githubusercontent.com/assets/5113964/8823857/4d0fa65a-3068-11e5-8113-c9ff4b6584a3.jpg)

**Comment by budnix** [comment]
I think that this bug can be already resolved. @jiteshvassa could you check this updated demo at http://jsfiddle.net/27jxhp7w/1/? I attached there Handsontable developer build which can by released tommorow.

**Comment by jiteshvassa** [comment]
@budnix I've just tried the demo but unfortunately seem to get the same behavior.

![pagedown-v2](https://cloud.githubusercontent.com/assets/5113964/8824016/c89bb9b6-3069-11e5-862d-091ee55f810c.jpg)

**Comment by jiteshvassa** [comment]
This looks like a Chrome issues - I've just tried my demo in Firefox and that appears okay.

Some more info on the problem.  If you click on one of the items on the table so that it has focus and then page down in Chrome then it works fine.  However if you click on the blank area on the right hand side of the table so that the table doesn't have focus, then the rendering issue seems to occur.

**Comment by budnix** [comment]
Yes you are right. Tested on Win 7 with Chrome and I have the same issues as you.

**Comment by krzysztofspilka** [comment]
@jiteshvassa it is confirmed then, thanks for letting us know.

**Comment by AMBudnik** [comment]
I tried the steps: 

1. Chrome at Win7
2. `if you click on the blank area on the right hand side of the table, such that the table doesn't have focus`
3. Press <kbd>PgDn</kbd>

and the issue does not occur.
